### DISCUSSION TITLE
What is the minimum GPU memory. to run it? Can it run with 16G?

### DISCUSSION THREAD

**Discussion by wangyong860401** [PixArt-alpha/PixArt-alpha#6] [Q&A] [answered]
What is the minimum GPU memory. to run it? Can it run with 16G?

**Comment by lawrence-cj** [comment]
Yes. You can change the T5 Model and VAE Model onto CPU, then 16G GPU memory is enough for inference.

**Comment by yujincheng08** [comment, marked as answer]
According to our test, 11G is enough with the diffuser integration. Remember to turn on offloading.
